src/_community/index.md

```diff
@@ -1,13 +1,36 @@
 ---
 permalink: /community/
 title: Community
+description: Learn about Quilt's community spaces, and what we expect of our users.
 ---
 
 You can find the Quilt in the following places.
 
-- Our [community discord]({{ site.discord_community }})
-- Our [toolchain discord]({{ site.discord_toolchain }})
+- Our [community Discord]({{ site.discord_community }})
+- Our [toolchain Discord]({{ site.discord_toolchain }})
 - Our [GitHub organization](https://github.com/{{ site.github_username }})
 
 In all of these places you are expected to follow our [rules]({% link _community/rules.md %})
 and [Code of Conduct]({% link _community/code-of-conduct.md %}).
+
+# Methodology
+
+Quilt's official community spaces exist to address issues in the wider modding community as much as the rest of the 
+project does. While it's certainly true that no large project should be without official community spaces, we had 
+several specific intentions in mind while creating it:
+
+* To build inclusivity and diversity into policy from the beginning, creating a safe community for minority groups as
+  best we can
+* To create a plural-friendly space by including tools like [PluralKit](/community/pluralkit) from a very early stage
```

Comment:
Could we make "plural" link to the "Plurality" section of our website as well?

Reply:
The community pages aren't being touched in this PR, as the entire section needs to be redone.